docs/parameters.md

```diff
@@ -21,6 +21,15 @@
 | `"metrics_enable"` | `bool` | Flag enabling [metrics](https://docs.openvino.ai/2022.3/ovms_docs_metrics.html) endpoint on rest_port. |    
 | `"metrics_list"` | `string` | Comma separated list of [metrics](https://docs.openvino.ai/2022.3/ovms_docs_metrics.html). If unset, only default metrics will be enabled.|
 
+
+
+> **Note** : Specifying config_path is mutually exclusive with putting model parameters in the CLI
+https://github.com/openvinotoolkit/model_server/blob/main/docs/starting_server.md#serving-multiple-models
```

Comment:
Can we have a relative link, so we don't redirect user from docs site to github?